rfcs/text/0000-create-file-macho.md

```diff
@@ -0,0 +1,146 @@
+|   macho.cpu.byte_order               |   keyword  |   CPU byte order for the file.                                              |
+|   macho.cpu.subtype                  |   keyword  |   CPU subtype for the file.                                                 |
+|   macho.cpu.type                     |   keyword  |   CPU type for the file.                                                    |
+|   macho.header                      |   object   |   Header information for the file.                                          |
+|   macho.header.commands             |   object   |   Header load commands for the file.                                        |
+|   macho.header.commands.number      |   long     |   Number of load commands for the Mach-O header.                            |
+|   macho.header.commands.size        |   long     |   Size of load commands of the Mach-O header.                               |
+|   macho.header.commands.type        |   keyword  |   Type of the load commands for the Mach-O header.                          |
+|   macho.header.magic                |   keyword  |   Magic field of the Mach-O header.                                         |
+|   macho.header.flags                |   keyword  |   Flags set in the Mach-O header.                                           |
+|   macho.segments                     |   flattened   |   Segment information for the file.                                         |
+|   macho.segments.vmaddr              |   keyword  |   Memory address of this segment.                                           |
+|   macho.segments.name                |   keyword  |   Name of this segment.                                                     |
+|   macho.segments.vmsize              |   keyword  |   Memory size of this segment.                                              |
+|   macho.segments.fileoff             |   keyword  |   File offset of this segment.                                              |
+|   macho.segments.filesize            |   keyword  |   Amount of memory to map from the file.                                    |
+|   macho.segments.sections            |   object   |   Section information for the segment of the file.                          |
+|   macho.segments.sections.flags      |   keyword  |   Section flags for the segment of the file.                                |
+|   macho.segments.sections.name       |   keyword  |   Section name for the segment of the file.                                 |
+|   macho.segments.sections.type       |   keyword  |   Section type for the segment of the file.                                 |
+|   macho.segments.offset      |   keyword  |   Offset of the segment.                                             |
+|   macho.segment size     |   keyword  |   Segment limit size.                                            |
```

Comment:
can you clean this up? think it should be `macho.segments.size` and `segments` should be moved to `nested` both here and in the field definitions.

Reply:
Done